flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -199,11 +209,13 @@ protected Cluster buildCluster(Cluster.Builder builder) {
     public static CassandraContainer createCassandraContainer() {
         CassandraContainer cassandra = new CassandraContainer(DockerImageVersions.CASSANDRA_3);
         cassandra.withJmxReporting(false);
+        cassandra.withLogConsumer(LOG_CONSUMER);
```

Comment:
How excessive are the Cassandra logs? For Kafka, we had to control the log level to avoid increasing the collected log size too much? [1]

[1] https://github.com/apache/flink/blob/00b519dff4279b38a1d9500dd5e7937944966b33/flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/testutils/KafkaUtil.java#L58

Reply:
Good point ! I don't know, I need to check

Reply:
On a sequence like this:
- start the container
- insert several rows using Cassandra writer
- select these rows
I get this log with only INFO and WARN. It looks good to me:
`CompilerOracle: dontinline org/apache/cassandra/db/Columns$Serializer.deserializeLargeSubset (Lorg/apache/cassandra/io/util/DataInputPlus;Lorg/apache/cassandra/db/Columns;I)Lorg/apache/cassandra/db/Columns;
CompilerOracle: dontinline org/apache/cassandra/db/Columns$Serializer.serializeLargeSubset (Ljava/util/Collection;ILorg/apache/cassandra/db/Columns;ILorg/apache/cassandra/io/util/DataOutputPlus;)V
CompilerOracle: dontinline org/apache/cassandra/db/Columns$Serializer.serializeLargeSubsetSize (Ljava/util/Collection;ILorg/apache/cassandra/db/Columns;I)I
CompilerOracle: dontinline org/apache/cassandra/db/transform/BaseIterator.tryGetMoreContents ()Z
CompilerOracle: dontinline org/apache/cassandra/db/transform/StoppingTransformation.stop ()V
CompilerOracle: dontinline org/apache/cassandra/db/transform/StoppingTransformation.stopInPartition ()V
CompilerOracle: dontinline org/apache/cassandra/io/util/BufferedDataOutputStreamPlus.doFlush (I)V
CompilerOracle: dontinline org/apache/cassandra/io/util/BufferedDataOutputStreamPlus.writeExcessSlow ()V
CompilerOracle: dontinline org/apache/cassandra/io/util/BufferedDataOutputStreamPlus.writeSlow (JI)V
CompilerOracle: dontinline org/apache/cassandra/io/util/RebufferingInputStream.readPrimitiveSlowly (I)J
CompilerOracle: inline org/apache/cassandra/io/util/Memory.checkBounds (JJ)V
CompilerOracle: inline org/apache/cassandra/io/util/SafeMemory.checkBounds (JJ)V
CompilerOracle: inline org/apache/cassandra/utils/AsymmetricOrdering.selectBoundary (Lorg/apache/cassandra/utils/AsymmetricOrdering/Op;II)I
CompilerOracle: inline org/apache/cassandra/utils/AsymmetricOrdering.strictnessOfLessThan (Lorg/apache/cassandra/utils/AsymmetricOrdering/Op;)I
CompilerOracle: inline org/apache/cassandra/utils/ByteBufferUtil.compare (Ljava/nio/ByteBuffer;[B)I
CompilerOracle: inline org/apache/cassandra/utils/ByteBufferUtil.compare ([BLjava/nio/ByteBuffer;)I
CompilerOracle: inline org/apache/cassandra/utils/ByteBufferUtil.compareUnsigned (Ljava/nio/ByteBuffer;Ljava/nio/ByteBuffer;)I
CompilerOracle: inline org/apache/cassandra/utils/FastByteOperations$UnsafeOperations.compareTo (Ljava/lang/Object;JILjava/lang/Object;JI)I
CompilerOracle: inline org/apache/cassandra/utils/FastByteOperations$UnsafeOperations.compareTo (Ljava/lang/Object;JILjava/nio/ByteBuffer;)I
CompilerOracle: inline org/apache/cassandra/utils/FastByteOperations$UnsafeOperations.compareTo (Ljava/nio/ByteBuffer;Ljava/nio/ByteBuffer;)I
CompilerOracle: inline org/apache/cassandra/utils/vint/VIntCoding.encodeVInt (JI)[B
INFO  09:45:20 Configuration location: file:/etc/cassandra/cassandra.yaml
INFO  09:45:20 Node configuration:[allocate_tokens_for_keyspace=null; authenticator=AllowAllAuthenticator; authorizer=AllowAllAuthorizer; auto_bootstrap=true; auto_snapshot=true; batch_size_fail_threshold_in_kb=50; batch_size_warn_threshold_in_kb=5; batchlog_replay_throttle_in_kb=1024; broadcast_address=172.17.0.3; broadcast_rpc_address=172.17.0.3; buffer_pool_use_heap_if_exhausted=true; cas_contention_timeout_in_ms=1000; check_for_duplicate_rows_during_compaction=true; check_for_duplicate_rows_during_reads=true; client_encryption_options=<REDACTED>; cluster_name=Test Cluster; column_index_size_in_kb=64; commit_failure_policy=stop; commitlog_compression=null; commitlog_directory=null; commitlog_max_compression_buffers_in_pool=3; commitlog_periodic_queue_size=-1; commitlog_segment_size_in_mb=32; commitlog_sync=periodic; commitlog_sync_batch_window_in_ms=null; commitlog_sync_period_in_ms=10000; commitlog_total_space_in_mb=null; compaction_large_partition_warning_threshold_mb=100; compaction_throughput_mb_per_sec=16; concurrent_compactors=null; concurrent_counter_writes=32; concurrent_materialized_view_writes=32; concurrent_reads=32; concurrent_replicates=null; concurrent_writes=32; counter_cache_keys_to_save=2147483647; counter_cache_save_period=7200; counter_cache_size_in_mb=null; counter_write_request_timeout_in_ms=5000; cross_node_timeout=false; data_file_directories=[Ljava.lang.String;@5ef60048; disk_access_mode=auto; disk_failure_policy=stop; disk_optimization_estimate_percentile=0.95; disk_optimization_page_cross_chance=0.1; disk_optimization_strategy=ssd; dynamic_snitch=true; dynamic_snitch_badness_threshold=0.1; dynamic_snitch_reset_interval_in_ms=600000; dynamic_snitch_update_interval_in_ms=100; enable_drop_compact_storage=false; enable_materialized_views=true; enable_scripted_user_defined_functions=false; enable_user_defined_functions=false; enable_user_defined_functions_threads=true; encryption_options=<REDACTED>; endpoint_snitch=SimpleSnitch; file_cache_size_in_mb=512; gc_log_threshold_in_ms=200; gc_warn_threshold_in_ms=1000; hinted_handoff_disabled_datacenters=[]; hinted_handoff_enabled=true; hinted_handoff_throttle_in_kb=1024; hints_compression=null; hints_directory=null; hints_flush_period_in_ms=10000; incremental_backups=false; index_interval=null; index_summary_capacity_in_mb=null; index_summary_resize_interval_in_minutes=60; initial_token=null; inter_dc_stream_throughput_outbound_megabits_per_sec=200; inter_dc_tcp_nodelay=false; internode_authenticator=null; internode_compression=all; internode_recv_buff_size_in_bytes=null; internode_send_buff_size_in_bytes=null; key_cache_keys_to_save=2147483647; key_cache_save_period=14400; key_cache_size_in_mb=null; listen_address=172.17.0.3; listen_interface=null; listen_interface_prefer_ipv6=false; listen_on_broadcast_address=false; max_hint_window_in_ms=10800000; max_hints_delivery_threads=2; max_hints_file_size_in_mb=128; max_mutation_size_in_kb=null; max_streaming_retries=3; max_value_size_in_mb=256; memtable_allocation_type=heap_buffers; memtable_cleanup_threshold=null; memtable_flush_writers=null; memtable_heap_space_in_mb=null; memtable_offheap_space_in_mb=null; min_free_space_per_drive_in_mb=50; native_transport_flush_in_batches_legacy=true; native_transport_max_concurrent_connections=-1; native_transport_max_concurrent_connections_per_ip=-1; native_transport_max_concurrent_requests_in_bytes=-1; native_transport_max_concurrent_requests_in_bytes_per_ip=-1; native_transport_max_frame_size_in_mb=256; native_transport_max_negotiable_protocol_version=-2147483648; native_transport_max_threads=128; native_transport_port=9042; native_transport_port_ssl=null; num_tokens=256; otc_backlog_expiration_interval_ms=200; otc_coalescing_enough_coalesced_messages=8; otc_coalescing_strategy=TIMEHORIZON; otc_coalescing_window_us=200; partitioner=org.apache.cassandra.dht.Murmur3Partitioner; permissions_cache_max_entries=1000; permissions_update_interval_in_ms=-1; permissions_validity_in_ms=2000; phi_convict_threshold=8.0; range_request_timeout_in_ms=10000; read_request_timeout_in_ms=5000; repair_session_max_tree_depth=18; request_scheduler=org.apache.cassandra.scheduler.NoScheduler; request_scheduler_id=null; request_scheduler_options=null; request_timeout_in_ms=10000; role_manager=CassandraRoleManager; roles_cache_max_entries=1000; roles_update_interval_in_ms=-1; roles_validity_in_ms=2000; row_cache_class_name=org.apache.cassandra.cache.OHCProvider; row_cache_keys_to_save=2147483647; row_cache_save_period=0; row_cache_size_in_mb=0; rpc_address=0.0.0.0; rpc_interface=null; rpc_interface_prefer_ipv6=false; rpc_keepalive=true; rpc_listen_backlog=50; rpc_max_threads=2147483647; rpc_min_threads=16; rpc_port=9160; rpc_recv_buff_size_in_bytes=null; rpc_send_buff_size_in_bytes=null; rpc_server_type=sync; saved_caches_directory=null; seed_provider=org.apache.cassandra.locator.SimpleSeedProvider{seeds=172.17.0.3}; server_encryption_options=<REDACTED>; snapshot_before_compaction=false; snapshot_on_duplicate_row_detection=false; ssl_storage_port=7001; sstable_preemptive_open_interval_in_mb=50; start_native_transport=true; start_rpc=false; storage_port=7000; stream_throughput_outbound_megabits_per_sec=200; streaming_socket_timeout_in_ms=86400000; thrift_framed_transport_size_in_mb=15; thrift_max_message_length_in_mb=16; tombstone_failure_threshold=100000; tombstone_warn_threshold=1000; tracetype_query_ttl=86400; tracetype_repair_ttl=604800; trickle_fsync=false; trickle_fsync_interval_in_kb=10240; truncate_request_timeout_in_ms=60000; unlogged_batch_across_partitions_warn_threshold=10; user_defined_function_fail_timeout=1500; user_defined_function_warn_timeout=500; user_function_timeout_policy=die; windows_timer_interval=1; write_request_timeout_in_ms=2000]
INFO  09:45:20 DiskAccessMode 'auto' determined to be mmap, indexAccessMode is mmap
INFO  09:45:20 Global memtable on-heap threshold is enabled at 252MB
INFO  09:45:20 Global memtable off-heap threshold is enabled at 252MB
INFO  09:45:20 Hostname: 12510c2b6c64
INFO  09:45:20 JVM vendor/version: OpenJDK 64-Bit Server VM/1.8.0_312
INFO  09:45:20 Heap size: 1060372480/1060372480
INFO  09:45:20 Code Cache Non-heap memory: init = 2555904(2496K) used = 6143232(5999K) committed = 6160384(6016K) max = 251658240(245760K)
INFO  09:45:20 Metaspace Non-heap memory: init = 0(0K) used = 16229384(15849K) committed = 16646144(16256K) max = -1(-1K)
INFO  09:45:20 Compressed Class Space Non-heap memory: init = 0(0K) used = 2032032(1984K) committed = 2228224(2176K) max = 1073741824(1048576K)
INFO  09:45:20 Par Eden Space Heap memory: init = 107479040(104960K) used = 71008432(69344K) committed = 107479040(104960K) max = 107479040(104960K)
INFO  09:45:20 Par Survivor Space Heap memory: init = 13369344(13056K) used = 0(0K) committed = 13369344(13056K) max = 13369344(13056K)
INFO  09:45:20 CMS Old Gen Heap memory: init = 939524096(917504K) used = 0(0K) committed = 939524096(917504K) max = 939524096(917504K)
INFO  09:45:20 Classpath: /etc/cassandra:/opt/cassandra/build/classes/main:/opt/cassandra/build/classes/thrift:/opt/cassandra/lib/ST4-4.0.8.jar:/opt/cassandra/lib/airline-0.6.jar:/opt/cassandra/lib/antlr-runtime-3.5.2.jar:/opt/cassandra/lib/apache-cassandra-3.0.25.jar:/opt/cassandra/lib/apache-cassandra-clientutil-3.0.25.jar:/opt/cassandra/lib/apache-cassandra-thrift-3.0.25.jar:/opt/cassandra/lib/asm-5.0.4.jar:/opt/cassandra/lib/cassandra-driver-core-3.0.1-shaded.jar:/opt/cassandra/lib/commons-cli-1.1.jar:/opt/cassandra/lib/commons-codec-1.2.jar:/opt/cassandra/lib/commons-lang3-3.1.jar:/opt/cassandra/lib/commons-math3-3.2.jar:/opt/cassandra/lib/compress-lzf-0.8.4.jar:/opt/cassandra/lib/concurrentlinkedhashmap-lru-1.4.jar:/opt/cassandra/lib/disruptor-3.0.1.jar:/opt/cassandra/lib/ecj-4.4.2.jar:/opt/cassandra/lib/guava-18.0.jar:/opt/cassandra/lib/high-scale-lib-1.0.6.jar:/opt/cassandra/lib/jackson-core-asl-1.9.2.jar:/opt/cassandra/lib/jackson-mapper-asl-1.9.2.jar:/opt/cassandra/lib/jamm-0.3.0.jar:/opt/cassandra/lib/javax.inject-1.jar:/opt/cassandra/lib/jbcrypt-0.3m.jar:/opt/cassandra/lib/jcl-over-slf4j-1.7.7.jar:/opt/cassandra/lib/jna-4.2.2.jar:/opt/cassandra/lib/joda-time-2.4.jar:/opt/cassandra/lib/json-simple-1.1.jar:/opt/cassandra/lib/libthrift-0.9.2.jar:/opt/cassandra/lib/log4j-over-slf4j-1.7.7.jar:/opt/cassandra/lib/logback-classic-1.1.3.jar:/opt/cassandra/lib/logback-core-1.1.3.jar:/opt/cassandra/lib/lz4-1.3.0.jar:/opt/cassandra/lib/metrics-core-3.1.0.jar:/opt/cassandra/lib/metrics-jvm-3.1.0.jar:/opt/cassandra/lib/metrics-logback-3.1.0.jar:/opt/cassandra/lib/netty-all-4.0.44.Final.jar:/opt/cassandra/lib/ohc-core-0.4.3.jar:/opt/cassandra/lib/ohc-core-j8-0.4.3.jar:/opt/cassandra/lib/reporter-config-base-3.0.0.jar:/opt/cassandra/lib/reporter-config3-3.0.0.jar:/opt/cassandra/lib/sigar-1.6.4.jar:/opt/cassandra/lib/slf4j-api-1.7.7.jar:/opt/cassandra/lib/snakeyaml-1.11.jar:/opt/cassandra/lib/snappy-java-1.1.1.7.jar:/opt/cassandra/lib/stream-2.5.2.jar:/opt/cassandra/lib/thrift-server-0.3.7.jar:/opt/cassandra/lib/jsr223/*/*.jar::/opt/cassandra/lib/jamm-0.3.0.jar
INFO  09:45:20 JVM Arguments: [-Dcassandra.skip_wait_for_gossip_to_settle=0, -Dcassandra.initial_token=0, -Xloggc:/opt/cassandra/logs/gc.log, -XX:+UseParNewGC, -XX:+UseConcMarkSweepGC, -XX:+CMSParallelRemarkEnabled, -XX:SurvivorRatio=8, -XX:MaxTenuringThreshold=1, -XX:CMSInitiatingOccupancyFraction=75, -XX:+UseCMSInitiatingOccupancyOnly, -XX:CMSWaitDuration=10000, -XX:+CMSParallelInitialMarkEnabled, -XX:+CMSEdenChunksRecordAlways, -XX:+CMSClassUnloadingEnabled, -XX:+PrintGCDetails, -XX:+PrintGCDateStamps, -XX:+PrintHeapAtGC, -XX:+PrintTenuringDistribution, -XX:+PrintGCApplicationStoppedTime, -XX:+PrintPromotionFailure, -XX:+UseGCLogFileRotation, -XX:NumberOfGCLogFiles=10, -XX:GCLogFileSize=10M, -Xms1024M, -Xmx1024M, -Xmn128M, -ea, -Xss256k, -XX:+AlwaysPreTouch, -XX:-UseBiasedLocking, -XX:StringTableSize=1000003, -XX:+UseTLAB, -XX:+ResizeTLAB, -XX:+PerfDisableSharedMem, -XX:CompileCommandFile=/etc/cassandra/hotspot_compiler, -javaagent:/opt/cassandra/lib/jamm-0.3.0.jar, -XX:+UseThreadPriorities, -XX:ThreadPriorityPolicy=42, -XX:+HeapDumpOnOutOfMemoryError, -Djava.net.preferIPv4Stack=true, -Dcassandra.jmx.local.port=7199, -XX:+DisableExplicitGC, -Djava.library.path=/opt/cassandra/lib/sigar-bin, -Dcassandra.libjemalloc=/usr/local/lib/libjemalloc.so, -XX:OnOutOfMemoryError=kill -9 %p, -Dlogback.configurationFile=logback.xml, -Dcassandra.logdir=/opt/cassandra/logs, -Dcassandra.storagedir=/opt/cassandra/data, -Dcassandra-foreground=yes]
WARN  09:45:20 Unable to lock JVM memory (ENOMEM). This can result in part of the JVM being swapped out, especially with mmapped I/O enabled. Increase RLIMIT_MEMLOCK or run Cassandra as root.
INFO  09:45:20 jemalloc seems to be preloaded from /usr/local/lib/libjemalloc.so
WARN  09:45:20 JMX is not enabled to receive remote connections. Please see cassandra-env.sh for more info.
INFO  09:45:20 Initializing SIGAR library
WARN  09:45:20 Cassandra server running in degraded mode. Is swap disabled? : false,  Address space adequate? : true,  nofile limit adequate? : true, nproc limit adequate? : true 
WARN  09:45:20 Maximum number of memory map areas per process (vm.max_map_count) 65530 is too low, recommended value: 1048575, you can change it with sysctl.
WARN  09:45:20 Directory /opt/cassandra/data/data doesn't exist
WARN  09:45:20 Directory /opt/cassandra/data/commitlog doesn't exist
WARN  09:45:20 Directory /opt/cassandra/data/saved_caches doesn't exist
WARN  09:45:20 Directory /opt/cassandra/data/hints doesn't exist
INFO  09:45:20 Initializing system.IndexInfo
INFO  09:45:21 Initializing system.batches
INFO  09:45:21 Initializing system.paxos
INFO  09:45:21 Initializing system.local
INFO  09:45:21 Initializing system.peers
INFO  09:45:21 Initializing system.peer_events
INFO  09:45:21 Initializing system.range_xfers
INFO  09:45:21 Initializing system.compaction_history
INFO  09:45:21 Initializing system.sstable_activity
INFO  09:45:21 Initializing system.size_estimates
INFO  09:45:21 Initializing system.available_ranges
INFO  09:45:21 Initializing system.views_builds_in_progress
INFO  09:45:21 Initializing system.built_views
INFO  09:45:21 Initializing system.hints
INFO  09:45:21 Initializing system.batchlog
INFO  09:45:21 Initializing system.schema_keyspaces
INFO  09:45:21 Initializing system.schema_columnfamilies
INFO  09:45:21 Initializing system.schema_columns
INFO  09:45:21 Initializing system.schema_triggers
INFO  09:45:21 Initializing system.schema_usertypes
INFO  09:45:21 Initializing system.schema_functions
INFO  09:45:21 Initializing system.schema_aggregates
INFO  09:45:21 Initializing system_schema.keyspaces
INFO  09:45:21 Initializing system_schema.tables
INFO  09:45:21 Initializing system_schema.columns
INFO  09:45:21 Initializing system_schema.triggers
INFO  09:45:21 Initializing system_schema.dropped_columns
INFO  09:45:21 Initializing system_schema.views
INFO  09:45:21 Initializing system_schema.types
INFO  09:45:21 Initializing system_schema.functions
INFO  09:45:21 Initializing system_schema.aggregates
INFO  09:45:21 Initializing system_schema.indexes
INFO  09:45:21 Initializing key cache with capacity of 50 MBs.
INFO  09:45:21 Initializing row cache with capacity of 0 MBs
INFO  09:45:21 Initializing counter cache with capacity of 25 MBs
INFO  09:45:21 Scheduling counter cache save to every 7200 seconds (going to save all keys).
INFO  09:45:21 Global buffer pool is enabled, when pool is exahusted (max is 512 mb) it will allocate on heap
INFO  09:45:21 Populating token metadata from system tables
INFO  09:45:21 Token metadata: 
INFO  09:45:21 Completed loading (0 ms; 1 keys) KeyCache cache
INFO  09:45:21 No commitlog files found; skipping replay
INFO  09:45:21 Populating token metadata from system tables
INFO  09:45:21 Token metadata: 
INFO  09:45:21 Cassandra version: 3.0.25
INFO  09:45:21 Thrift API version: 20.1.0
INFO  09:45:21 CQL supported versions: 3.4.0 (default: 3.4.0)
INFO  09:45:21 Initializing index summary manager with a memory pool size of 50 MB and a resize interval of 60 minutes
INFO  09:45:21 Loading persisted ring state
WARN  09:45:21 No host ID found, created 3b6760bc-b0ab-427b-9b74-f0f4e5a45414 (Note: This should happen exactly once per node).
INFO  09:45:21 Starting up server gossip
INFO  09:45:21 Starting Messaging Service on /172.17.0.3:7000 (eth0)
INFO  09:45:21 This node will not auto bootstrap because it is configured to be a seed node.
INFO  09:45:21 Saved tokens not found. Using configuration value: [0]
INFO  09:45:21 Initializing system_traces.events
INFO  09:45:21 Initializing system_traces.sessions
INFO  09:45:21 Initializing system_distributed.parent_repair_history
INFO  09:45:21 Initializing system_distributed.repair_history
INFO  09:45:21 Initializing system_auth.resource_role_permissons_index
INFO  09:45:21 Initializing system_auth.role_members
INFO  09:45:21 Initializing system_auth.role_permissions
INFO  09:45:21 Initializing system_auth.roles
INFO  09:45:22 Netty using native Epoll event loop
INFO  09:45:22 Using Netty Version: [netty-buffer=netty-buffer-4.0.44.Final.452812a, netty-codec=netty-codec-4.0.44.Final.452812a, netty-codec-haproxy=netty-codec-haproxy-4.0.44.Final.452812a, netty-codec-http=netty-codec-http-4.0.44.Final.452812a, netty-codec-socks=netty-codec-socks-4.0.44.Final.452812a, netty-common=netty-common-4.0.44.Final.452812a, netty-handler=netty-handler-4.0.44.Final.452812a, netty-tcnative=netty-tcnative-1.1.33.Fork26.142ecbb, netty-transport=netty-transport-4.0.44.Final.452812a, netty-transport-native-epoll=netty-transport-native-epoll-4.0.44.Final.452812a, netty-transport-rxtx=netty-transport-rxtx-4.0.44.Final.452812a, netty-transport-sctp=netty-transport-sctp-4.0.44.Final.452812a, netty-transport-udt=netty-transport-udt-4.0.44.Final.452812a]
INFO  09:45:22 Starting listening for CQL clients on /0.0.0.0:9042 (unencrypted)...
INFO  09:45:22 Not starting RPC server as requested. Use JMX (StorageService->startRPCServer()) or nodetool (enablethrift) to start it
INFO  09:45:22 Startup complete
INFO  09:45:25 Create new Keyspace: KeyspaceMetadata{name=flink, params=KeyspaceParams{durable_writes=true, replication=ReplicationParams{class=org.apache.cassandra.locator.SimpleStrategy, replication_factor=1}}, tables=[], views=[], functions=[], types=[]}
INFO  09:45:26 Create new table: org.apache.cassandra.config.CFMetaData@7c8e4956[cfId=05a8e8e0-801f-11ec-b935-f58bd8d3766a,ksName=flink,cfName=flink_initial,flags=[COMPOUND],params=TableParams{comment=, read_repair_chance=0.0, dclocal_read_repair_chance=0.1, bloom_filter_fp_chance=0.01, crc_check_chance=1.0, gc_grace_seconds=864000, default_time_to_live=0, memtable_flush_period_in_ms=0, min_index_interval=128, max_index_interval=2048, speculative_retry=99PERCENTILE, caching={'keys' : 'ALL', 'rows_per_partition' : 'NONE'}, compaction=CompactionParams{class=org.apache.cassandra.db.compaction.SizeTieredCompactionStrategy, options={min_threshold=4, max_threshold=32}}, compression=org.apache.cassandra.schema.CompressionParams@d6c64903, extensions={}},comparator=comparator(),partitionColumns=[[] | [batch_id counter]],partitionKeyColumns=[ColumnDefinition{name=id, type=org.apache.cassandra.db.marshal.UTF8Type, kind=PARTITION_KEY, position=0}],clusteringColumns=[],keyValidator=org.apache.cassandra.db.marshal.UTF8Type,columnMetadata=[ColumnDefinition{name=counter, type=org.apache.cassandra.db.marshal.Int32Type, kind=REGULAR, position=-1}, ColumnDefinition{name=batch_id, type=org.apache.cassandra.db.marshal.Int32Type, kind=REGULAR, position=-1}, ColumnDefinition{name=id, type=org.apache.cassandra.db.marshal.UTF8Type, kind=PARTITION_KEY, position=0}],droppedColumns={},triggers=[],indexes=[]]
INFO  09:45:26 Initializing flink.flink_initial
INFO  09:45:27 Create new table: org.apache.cassandra.config.CFMetaData@3b1c03c0[cfId=064c06b0-801f-11ec-b935-f58bd8d3766a,ksName=flink,cfName=flink_1976008709,flags=[COMPOUND],params=TableParams{comment=, read_repair_chance=0.0, dclocal_read_repair_chance=0.1, bloom_filter_fp_chance=0.01, crc_check_chance=1.0, gc_grace_seconds=864000, default_time_to_live=0, memtable_flush_period_in_ms=0, min_index_interval=128, max_index_interval=2048, speculative_retry=99PERCENTILE, caching={'keys' : 'ALL', 'rows_per_partition' : 'NONE'}, compaction=CompactionParams{class=org.apache.cassandra.db.compaction.SizeTieredCompactionStrategy, options={min_threshold=4, max_threshold=32}}, compression=org.apache.cassandra.schema.CompressionParams@d6c64903, extensions={}},comparator=comparator(),partitionColumns=[[] | [batch_id counter]],partitionKeyColumns=[ColumnDefinition{name=id, type=org.apache.cassandra.db.marshal.UTF8Type, kind=PARTITION_KEY, position=0}],clusteringColumns=[],keyValidator=org.apache.cassandra.db.marshal.UTF8Type,columnMetadata=[ColumnDefinition{name=counter, type=org.apache.cassandra.db.marshal.Int32Type, kind=REGULAR, position=-1}, ColumnDefinition{name=batch_id, type=org.apache.cassandra.db.marshal.Int32Type, kind=REGULAR, position=-1}, ColumnDefinition{name=id, type=org.apache.cassandra.db.marshal.UTF8Type, kind=PARTITION_KEY, position=0}],droppedColumns={},triggers=[],indexes=[]]
INFO  09:45:28 Initializing flink.flink_1976008709
INFO  09:45:32 Created default superuser role 'cassandra'
`